roofit/RDataFrameHelpers/inc/LinkDef.h

```diff
@@ -0,0 +1,6 @@
+#ifdef __CINT__
+
+#pragma link off all globals;
+#pragma link off all classes;
+
+#endif
```

Comment:
is this different from not having a LinkDef at all?

Reply:
You cannot not have a LinkDef IIUC. CMake just went crazy when I tried.

Unless you know how to trick it? @eguiraud

Reply:
there are a few invocations of `ROOT_STANDARD_LIBRARY_PACKAGE` without a linkdef argument around the repo, I don't know more than that. See e.g. graf3d/x3d/CMakeLists.txt.